### ISSUE TITLE
EnumString Keep/Rename/Remove

### ISSUE THREAD

**Issue by jplatte** [Peternator7/strum#136]
Is there a reason there wasn't an issue with this title already? If it was missing, feel free to replace the issue description :)

**Comment by jplatte** [comment]
This one I think should be kept but renamed. Probably to `FromStr` since that is what the macro generates an impl for.

**Comment by linclelinkpart5** [comment]
@jplatte I agree with your suggestion, I find myself confusing this with `AsRefStr`/`ToString` often. Having `From` in the name would help with indicating that this is the &str -> enum conversion, as opposed to enum -> &str/String conversion.
